src/quemb/molbe/chemfrag.py

```diff
@@ -196,6 +197,9 @@ def from_cartesian(
               to change the radius of individual elements, e.g. :python:`{"C": 1.5}`.
 
             The keyword is mutually exclusive with :python:`bonds_atoms`.
+        modify_atom_data :
+            To change the van der Waals radius of one or more specific atoms, pass a
+            dictionary that looks like {index1: 1.5}.
```

Comment:
I would use
```
:python:`{index1: 1.5}`
```
here and at other occurences, where it should be rendered as code